commons/src/main/java/com/powsybl/commons/report/Formatter.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.commons.report;
+
+/**
+ * @author Florian Dupuy {@literal <florian.dupuy at rte-france.com>}
+ */
+@FunctionalInterface
+public interface Formatter {
```

Comment:
Maybe we can name it ReportFormatter or MessageFormatter to avoid confusion with technical / JDK formatting classes ?